hello-1/hello.pro

```diff
@@ -2,6 +2,7 @@ TEMPLATE = lib
 TARGET = hello
 
 CONFIG += static conan_basic_setup
+CONFIG += debug

```

Comment:
This CONFIG has to match the conan setting ``build_type``. Now it is just hardcoded, but should be taken into account.